src/backend/distributed/Makefile

```diff
@@ -36,9 +36,37 @@ OBJS += \
 
 all: cdc
 
+setup_cdc_build_dir_copy_files:
+	mkdir -p $(DECODER_BUILD_DIR)
+	$(eval SRC_DIR = $(citus_abs_srcdir)/cdc)
+	$(eval DEST_DIR = $(DECODER_BUILD_DIR))
+	@for file in $(SRC_DIR)/*.c $(SRC_DIR)/*.h $(SRC_DIR)/Makefile; do \
+		if [ -f $$file ]; then \
+            if ! diff -q $$file $(DEST_DIR)/$$(basename $$file); then \
+                cp $$file $(DEST_DIR)/$$(basename $$file); \
+            fi \
+        fi \
+    done
+
+cdc_build_dir:
+	$(eval DECODER_BUILD_DIR = $(citus_abs_srcdir)/build/cdc-$(DECODER))
+	$(MAKE) DECODER_BUILD_DIR=$(DECODER_BUILD_DIR) setup_cdc_build_dir_copy_files
+	$(MAKE) DECODER=$(DECODER) citus_top_builddir="../../../../.." \
+		-C $(DECODER_BUILD_DIR)
+
+cdc_clean_dir:
+	$(eval DECODER_BUILD_DIR = $(citus_abs_srcdir)/build/cdc-$(DECODER))
+	$(MAKE) DECODER=$(DECODER) citus_top_builddir="../../../../.." \
+		-C $(DECODER_BUILD_DIR) clean
+
+cdc_install_dir:
```

Comment:
minor: seems like our make targets usually use dashes instead of underscores

Reply:
Done